fobi_custom/plugins/form_elements/fields/observational/race_observational/forms.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from fobi.base import BaseFormFieldPluginForm, get_theme
+
+theme = get_theme(request=None, as_instance=True)
+
+
+class RaceObservationalForm(forms.Form, BaseFormFieldPluginForm):
+    """RaceObservationalForm."""
+
+    plugin_data_fields = [
+        ("label", "How many people of each perceived category do you see? (select all that apply)"),
```

Comment:
It'd be nice to move `Select all that apply` to be the default `help_text` for this question, but I couldn't figure out how to prepopulate it after messing with it for a few minutes (I tried the `initial` kwarg in the `CharField` below, but that didn't seem to work).

Reply:
I'll leave a comment where this should go!

Reply:
To maintain consistency, I'd cut "(select all that apply)" from here